build/rpm/GNUmakefile

```diff
@@ -123,5 +123,9 @@ pcp.spec: pcp.spec.in
 	    -e's|@make@|$(PCP_MAKE_PROG)|g' <$? >$@
 	./do-maybe-requires
 
+systemd-sysuser-configs:
+	$(MAKE) $(MAKEOPTS) -C $(TOPDIR)/src/include pcp.user.install
+	$(MAKE) $(MAKEOPTS) -C $(TOPDIR)/qa pcpqa.user
+
```

Comment:
Hmm, I don't follow ... these files have already been built elsewhere in the source tree haven't they?  (I'm probably missing something, but if not - why do this here as well?)